content/handbook/sales/gainsight/_index.md

```
@@ -0,0 +1,110 @@
+Gainsightは顧客のライフサイクルのいくつかの異なる領域で役立ちます。いくつかのハイライトには以下が含まれます:
+
+- 効率: 統合されたアカウントビュー(Book of Business、アカウント)、製品分析、Zendesk統合、メモ、コラボプロジェクト、to-do
+- 一貫性: 顧客ライフサイクルプロセスを確立し、エンゲージメントを管理・追跡
+- 可視性: ヘルススコア、リスク、採用、データビジュアル
+- 自動化: デジタルジャーニーのプロセス、採用、イネーブルメント
+- メトリックと分析: ステージ採用、顧客のヘルス、time-to-value、製品利用データ
+- ネットリテンションを成長させる: サクセスプラン主導のエンゲージメント、エクスパンドプレイ
+
+## はじめに
+
+Gainsightにアクセスする方法は2つあります: [Salesforce経由](#access-through-salesforce)(強く推奨)、および[直接ログイン](#logging-in-directly)。
```

markdownlint-cli2 (0.22.1)

[warning] Link fragments should be valid

(MD051, link-fragments)